core/src/components/range/range.md.scss

```diff
@@ -177,11 +177,7 @@
      */
     @include position(null, null, -1px, 50%);
     @include margin-horizontal(-13px, null);
-
-    @include multi-dir() {
-      /* stylelint-disable-next-line property-disallowed-list */
-      border-radius: 50% 50% 50% 0;
-    }
+    @include border-radius(50% 50% 50% 0);
```

Comment:
Is separating arguments in a mixin optional within SASS? For example all our other usages comma separate the individual arguments, but I see here we aren't needing to do that.

Reply:
This was unintentional! It actually ends up assigning the entire chunk to `border-radius` if you only pass one argument to it:

```scss
.test-border-radius-one-argument {
  @include border-radius(50% 50% 50% 0);
}

.test-border-radius-multiple-arguments {
  @include border-radius(50%, 50%, 50%, 0);
}
```

Becomes:

```css
.test-border-radius-one-argument {
  border-radius: 50% 50% 50% 0;
}

.test-border-radius-multiple-arguments {
  border-start-start-radius: 50%;
  border-start-end-radius: 50%;
  border-end-end-radius: 50%;
  border-end-start-radius: 0;
}
```

I have updated it to be comma separated: https://github.com/ionic-team/ionic-framework/pull/29002/commits/86962bf11bf916f958124b1f56f3625abb2bee52